content/panorama/scripts/team_select.js

```diff
@@ -30,6 +30,7 @@ function hidePregameUI () {
 function showPregameUI () {
   FindDotaHudElement('PreGame').style.opacity = 1;
   FindDotaHudElement('PreGame').style.visibility = 'visible';
+  FindDotaHudElement('PreGame').FindChildTraverse( 'GameModeLabel' ).text = "Open Angel Arena";
```

Comment:
I don't really think that's a good place to put it; not obvious. Regardless, we need to put in the localization stuff and call `$.Localize` instead of hardcoding the text unless we don't intend to translate that bit.